earth_enterprise/src/common/tests/SConscript

```diff
@@ -50,8 +50,16 @@ map(lambda test: env.test(test, [test + '.cc'],
     ['etencoder_tests',
      ])
 
+testdir = os.path.join(env.exportdirs['bin'],'tests')
+env.copyfile(testdir, 'RunAllTests.pl')
 
-env.copyfile(os.path.join(env.exportdirs['bin'],'tests'),
-             'RunAllTests.pl')
+# Drop in the test data
+env.Command(
```

Comment:
@andreisel I don't really understand why this is necessary. The command to copy the RunAllTests.pl file works fine the way it is. And I think moving it underneath the env.Command call just makes it a little more confusing in my opinion.